iphone/Classes/TiUIWindowProxy.h

```diff
@@ -27,6 +27,8 @@
 - (void)refreshBackButton;
 - (void)updateNavBar;
 - (void)boot:(BOOL)timeout args:(id)args;
+@property (nonatomic, assign) TiViewProxy *safeAreaViewProxy;
```

Comment:
Add one tmpty line for readability. Also ensure to format everything with our clang-format